proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and accessing OCI Runtime and Image specification JSON.
```

Comment:
Provide code for parsing and accessing JSON versions of OCI Runtime and Image specs. The first implementation will be a C library automatically generated directly from the OCI Runtime and Image specifications.